### PR TITLE
merge type judgment into /std/util/types.ts

### PR DESCRIPTION
**Pull request by xiaoxintang**

### CODE REVIEW

**Comment by nayeemrmn** [comment]
> > You have to specify the input modules at the end of your command or it'll try to format everything in there. Working as expected.
> 
> sorry, my fault,i just follow the `/std/prettier/README.md`,by the way , can you tell me why i can‘t pass the [CI](https://github.com/denoland/deno/pull/3517/checks?check_run_id=353767249)

Run `deno fmt std/**/*.ts`.

**Comment by kevinkassimo** [comment]
@myzle Please refrain from verbal attacks on others. I do not have the permission to moderate your comment but you are not being helpful.

We might need a basic CODE_OF_CONDUCT.md.

**Comment by xiaoxintang** [comment]
> @myzle 
it's to hard for me,Third failure

**Comment by xiaoxintang** [comment]
thanks,it works

**Comment by xiaoxintang** [comment]
@kevinkassimo @nayeemrmn 
help

File: std/flags/mod.ts [hunk 1]
```diff
@@ -36,12 +37,6 @@ function get<T>(obj: { [s: string]: T }, key: string): T | undefined {
   }
 }
 
-function isNumber(x: unknown): boolean {
```
Comment (kevinkassimo) [inline std/flags/mod.ts]: I believe you should keep this implementation. This is causing test failures.

`isNumber` here also accept string argument and test if it is a string representation of some number

Comment (xiaoxintang) [reply std/flags/mod.ts]: oh,sorry.but i think there is no strict enough if we use string as a number
> in  `std/util/types_test.ts` line 61 , i have  test Hexadecimal
> and i test`0.2e3 0.2e-3 -3.2E3` in chrome'console by `typeof`

**Pull request closed without merging**